.github/workflows/gen-docs.yml

```diff
@@ -0,0 +1,36 @@
+name: Generate Documentation
+
+on: [release]
+
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  build:
+
+    runs-on: macOS-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Install Jazzy
+      run: gem install jazzy
+    - name: Build
+      run: |
+        swift package generate-xcodeproj
+        jazzy --clean
+    - name: Commit
+      run: |
+        remote_repo="https://x-access-token:${GITHUB_TOKEN}@github.com/${GITHUB_REPOSITORY}.git"
+        remote_branch="gh-pages"
```

Comment:
Maybe we do env variables in all CAPS?
Further we could define env variables in the env section.

```yaml
- name: Commit
  env:
    REMOTE_REPO: "https://x-access-token:${GITHUB_TOKEN}@github.com/${GITHUB_REPOSITORY}.git"
    REMOTE_BRANCH: "gh-pages"
  run: |
    ...
```

Reply:
done, plus moved GITHUB_TOKEN here

Reply:
Tried moving the environment vars to env: section. It doesn't like expanding environment vars in that section so $REMOTE_REPO was not expanding $GITHUB_TOKEN and $GITHUB_REPOSITORY.